packages/react-component-playground/test/components/component-playground/selected-fixture-and-editor/events/handlers.js

```diff
@@ -62,13 +62,11 @@ describe(`ComponentPlayground (${FIXTURE}) Events Handlers`, function() {
     });
   });
 
-  it('should set splitpane onChange to localstorage lib set', function() {
-    expect(component.refs.splitPane.props.onChange())
-      .to.equal(localStorageLib.set('splitPos'));
-  });
+  it('should have split-pane position in local storage on change', function() {
+    var size = 10;
 
-  it('should set splitpane defaultSize to localstorage lib get', function() {
```

Comment:
Oh, I thought you meant testing the props isn't the way to go, so I added a new test for the onChange effects.

The two tests are only asserting that props are sent correctly. The new one asserts that `splitPane` actually calls that callback, but I guess this is `react-split-pane`'s job, so we don't need to test that.

I'm going back to the two initial tests now.